host_os.py

```diff
@@ -32,11 +34,36 @@
     'build-iso': build_iso,
 }
 
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
+    """
+
+    cmd = "rpm -q %s" % package_name
+    try:
+        run_command(cmd, shell=True)
+    except SubprocessError as e:
+        # grep returns 1 when search string is not found and 2 if an error occurred
+        if e.returncode == 1:
+            return False
+        else:
+            raise
+
+    return True
+
 
 if __name__ == '__main__':
     CONF = config.setup_default_config()
     subcommand = CONF.get('default').get('subcommand')
 
+    PACKAGE_DEPENDENCIES_FILE_PATH = "rpm_requirements.txt"
+    package_dependencies = open(PACKAGE_DEPENDENCIES_FILE_PATH).readlines()
+    package_dependencies = [x.strip('\n') for x in package_dependencies]
+    for package in package_dependencies:
+        if not is_package_installed(package):
+            print("%s package should be installed before running this script" % package)
+            sys.exit(4)
+
```

Comment:
The `PACKAGE_DEPENDENCIES_FILE_PATH` is a huge variable name and it is used only once.  I'd use a variable if its value comes from a configuration file.  Since its value is hard-coded here, I don't see the point for a variable.

If the system does not have all of the rpm requirements, `host_os.py` will exit with the same error every time user tries to run it until all packages are installed.  I guess it would be better for user to know all package names that there are missing on his system in one shot.

What about defining an error code when packages are not installed, similar to what has been proposed in pull 143?

I'd also use list-comprehension to simplify this block.  For example:

```python
with open("reqs.txt") as f:
    pkg_reqs = [l.strip('\n') for l in f.readlines()]

not_installed = [p for p in pkg_reqs if not is_package_installed(p)]

if not_installed:
    print("The following packages should be installed before running this "
          "script:\n%s" % " ".join(not_installed))
    sys.exit(DEPS_NOT_INSTALLED_ERROR)
```